workspaces/cli-shared/src/httptoolkit-capturing-proxy.ts

```diff
@@ -204,6 +205,11 @@ export class HttpToolkitCapturingProxy {
           })
         );
 
+        if (!isValidHttpInteraction(sample)) {
+          developerDebugLogger('Filtering out invalid sample', sample);
+          return;
+        }
```

Comment:
I wanted to add a sentry trigger here, but I don't think our proxy (which runs through api start) actually has sentry instrumented. I'll mark this down as a follow up